draft-ietf-acme-acme.md

```diff
@@ -1867,7 +1867,7 @@ type (required, string):
 
 token (required, string):
 : A random value that uniquely identifies the challenge.  This value MUST have
-at least 128 bits of entropy, in order to prevent an attacker from guessing it.
+at least 128 bits of entropy.
```

Comment:
@zjs Good catch - thanks! Updated in https://github.com/ietf-wg-acme/acme/pull/284/commits/1eece0d781c62e5f047ea292f7838851f9d4586d